### PR TITLE
Linting yamls with yamllint 🏷

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Next update of plumbing will enable `yamllint` check, so this make
sure we have linted yaml before 👼

Related to tektoncd/catalog#101 (not closing it yet as the CI is not hooked up, needs tektoncd/plumbing#111 and a prow config update :stuck_out_tongue: 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: .yamllint [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  truthy:
+    level: warning
+
+  # accept both     key:
+  #                   - item
+  #
+  # and             key:
+  #                 - item
+  indentation:
+    indent-sequences: whatever
```
Comment (sbwsg) [inline .yamllint]: hah.  `whatever`

Comment (vdemeester) [reply .yamllint]:
```
indent-sequences defines whether block sequences should be indented or not (when in a mapping, this indentation is not mandatory – some people perceive the - as part of the indentation). Possible values: true, false, whatever and consistent. consistent requires either all block sequences to be indented, or none to be. whatever means either indenting or not indenting individual block sequences is OK.
```

Comment (vdemeester) [reply .yamllint]: The name is a bit scary :stuck_out_tongue_closed_eyes:

Comment (vdemeester) [reply .yamllint]: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation

**Pull request merged**